### PR TITLE
Added Organization commands

### LINKED ISSUE #7: Commands: Organization
- [ ] `vf-admin org get <id>`
- [ ] `vf-admin org list`
- [ ] `vf-admin org add`
- [ ] `vf-admin org update <id>`
- [ ] `vf-admin org remove <id>`

The issue includes all of the commands mentioned above. For all of them except the last command, print out the response on a table. Any values from the HTTP body request should be set as a key (check if schema sets it as required or not). The `list` command may utilize `page` and `size` as keys as well but they are not required (page is set to 1 by default and size is set to 12 by default). *Note that pagination may have not been implemented yet in the API.*

Don't forget to document all commands carefully and run `make docs` or `go run ./scripts/gen-docs/main.go` to generate documentation on the commands you created.

Before committing, make sure you have `pre-commit` hooks installed. If not, run `gofmt`, `go lint` and `go vet` by yourself.

💉 💉 💉

### PR DESCRIPTION
**Pull request by EricStG**
Added commands for organization management

Closes #7

### CODE REVIEW

File: internal/utils/string.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package utils
+
+// CoalesceString returns empty string on null, otherwise returns the value
+func CoalesceString(value *string) string {
```
Comment (EricStG) [inline internal/utils/string.go:4]: I added this little func to help me avoid splattering the code with if/else. Let me know what you think

File: cmd/organization.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+func init() {
+	RootCmd.AddCommand(organizationCmd)
+
+	// Get command
+	organizationCmd.AddCommand(organizationGetCmd)
+
+	// List command
+	organizationCmd.AddCommand(organizationListCmd)
```
Comment (alvintangz) [inline cmd/organization.go:87]: We're gonna take page and size as query params later for the API list endpoints fyi

File: internal/cmdrun/organization/list/list.go [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	"strconv"
+	"vf-admin/internal/utils"
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+)
+
+// CmdRunE is what's executed when running `vf-admin organization list`
+func CmdRunE(cmd *cobra.Command, args []string) error {
+	// Create the API client using the authentication key for requests
```
Comment (alvintangz) [inline internal/cmdrun/organization/list/list.go]: Prob copied my code - should be `// Create the API client`, ignore the auth key part

File: internal/utils/string.go (hunk 1 above)
Comment (alvintangz) [reply internal/utils/string.go:4]: It's fine

**Review by alvintangz — COMMENTED** [review]
It's easy as you say it is. Haven't tested it but looks good.

At this pace, I'm wondering if you can spend some time thinking about how to optimize the code instead. Maybe create a function for each CRUD operation take takes in the client func, response type, body (if applicable), etc... because there's a lot of copying and pasting going around here. Maybe take a closer look at api/client.gen.go as it has more functions that take in more abstract things.

**Comment by EricStG** [comment]
I can look at reducing boilerplate code, but all I can think off right now isn't supported in golang 😭 
I'll see what I can come up with

**Review by alvintangz — APPROVED** [review]
Looks great! Cheers! 🍻

**Pull request merged**